Services/CO.CDP.EntityVerification.Tests/Persistence/DatabasePponRepositoryTest.cs

```diff
@@ -27,20 +27,24 @@ public async Task ItFindsSavedPpon()
     public void ItRejectsAnAlreadyKnownPponId()
     {
         using var repository = PponRepository();
-        var ppon = GivenPpon(pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
-        var anotherPpon = GivenPpon(pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
+        var ppon = GivenPpon("Acme Corp Ltd", pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
+        var anotherPpon = GivenPpon("Acme Corp Ltd", pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
 
         repository.Save(ppon);
 
         repository.Invoking(r => r.Save(anotherPpon))
             .Should().Throw<DuplicatePponException>();
     }
 
-    private static EntityVerification.Persistence.Ppon GivenPpon(string? pponId = null)
+    private static EntityVerification.Persistence.Ppon GivenPpon(string orgName, string? pponId = null)
     {
+        var ppon = GivenPpon("Acme Corp Ltd", pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
+        var newIdentifier = new Identifier { Ppon = ppon, LegalName = orgName, Scheme = "" };
```

Comment:
Something went wrong there. The ppon is its own entity. Identifier is an entity to represent all the other identifiers.